gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -211,9 +210,11 @@ def run_compute_info(mt: hl.MatrixTable) -> hl.Table:
             "release": mt.meta.release,
             "unrelated": ~mt.meta.sample_filters.relatedness_filters.related,
         },
+        n_partitions=None,
```

Comment:
did you skip the naive_coalesce here to make the join with the `set_long_AS_missing` versions of the annotations faster?

Reply:
I did it because for the really long step of computation, instead of using the full ~15,000 partitions, it was splitting it into only 5,000 partitions so I was worried about things possibly being slower and taking more memory.